src/styles.scss

```diff
@@ -215,3 +215,33 @@ mat-card.dashboard-widget {
 .mat-select-disabled .mat-select-value {
   color: black;
 }
+
+/* configuration for the mat-slide-toggle feature, used e.g. in the entity-list.component for the "Show All"
+option within the pagination of the tables */
```

Comment:
Can this be moved to `list-paginator.component.scss`? Otherwise this might influence other toggles components

Reply:
Yes, it has been moved.